.github/workflows/android-ci.yml

```diff
@@ -25,17 +25,11 @@ on:
 
 jobs:
```

Comment:
Should we keep just one Github Action file here? Does not have to be done in this PR. I think having `android-ci.yml` and `android-ci-pull.yml` both for nearly the same outcome is unnecessary. [Reusing GIthub Workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows) should simplify the amount of copy-pasta 🍝  here.

Reply:
For example, ccache setup and restore. This can easily be made to a reusable workflow with cache key passed as an input. Tired of seeing that copied and pasted everywhere like a plague.

Reply:
I agree, but yes let's do a separate PR for that.

Reply:
Created an issue for it: https://github.com/maplibre/maplibre-gl-native/issues/735